src/styles/blocks/container.scss

```
@@ -0,0 +1,40 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  max-width: 1199px;
+
+  // @media (min-width: 1200px) {
+  //   grid-template-columns: repeat(3, minmax(300px, 1fr));
+  // }
+
+  &__box {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    height: 300px;
+
+    &--color1 {
+      background-color: rgb(100%, 0%, 0%);
+    }
+  
+    &--color2 {
+      background-color: rgb(80%, 0%, 0%);
+    }
+  
+    &--color3 {
+      background-color: rgb(60%, 0%, 0%);
+    }
+  
+    &--color4 {
+      background-color: rgb(40%, 0%, 0%);
+    }
+  
+    &--color5 {
+      background-color: rgb(20%, 0%, 0%);
+    }
+  
+    &--color6 {
+      background-color: rgb(0%, 0%, 0%);
+    }
+  }
```

↯....Also 25 rows using sass you can write more shortest.
```suggestion
      @each $box-num, $color in $color-list {
    &__box:nth-child(#{$box-num}) {
      background-color: $color;
    }
```

---

agree